DcCore/DcCore/DC/Wrapper.swift

```diff
@@ -837,12 +848,8 @@ public class DcMsg {
         return Int(dc_msg_get_from_id(messagePointer))
     }
 
-    public lazy var fromContact: DcContact = {
-        DcContact(id: fromContactId)
```

Comment:
good to get rid of that:
- it is an expensive call that does not look as such
- it was never updated due to `lazy`